python/tests/spark/sql/codegen/test_torchhub_registry.py

```diff
@@ -0,0 +1,37 @@
+#  Copyright (c) 2021 Rikai Authors
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from pyspark.sql import SparkSession
+
+from rikai.contrib.torch.transforms.fasterrcnn_resnet50_fpn import (
+    OUTPUT_SCHEMA,
+)
+
+
+def test_create_model(rikai_spark: SparkSession):
+    # TODO: run ml_predict on resnet50
+    # rikai.contrib.torch.transforms.fasterrcnn_resnet50_fpn
+    # does not actually work for torchhub loaded model
```

Comment:
where to put torchhub transforms pre/post precessors, this needs to be dicuessed.